include/mconfig.h

```diff
@@ -157,6 +161,10 @@ struct MConfig {
 	int button_integrated;	// Is the button under the touchpad? 0 or 1
 	int button_expire;		// How long to consider a touch for button emulation. >= 0
 	int button_zones;		// Use button zones for emulation?
+	int button_zones_in_edge_bottom;	// Height of the button zones
+	int zones_button_1;		// Button to execute when clicking in the leftmost part of the zone (button_zones > 0 and zones_height > 0)
+	int zones_button_2;		// Button to execute when clicking right of zone_button1 part of the zone (button_zones > 0 and zones_height > 0)
+	int zones_button_3;		// Button to execute when clicking the rightmost part of the zone (button_zones > 0 and zones_height > 0)
```

Comment:
I also think the naming is wrong yet I'm not sure about this.   
It is possible to activate each `button_zone` separately in the config, and you can bind, lets say only `zones_button_3` to a left click like:
```
Option "ButtonZonesEnable" "true"
Option "ButtonZone1" "0"
Option "ButtonZone2" "0"
Option "ButtonZone3" "1"
```
If we rename `zones_button_3` to `zone_right_button`, it will be misleading since it can be triggered on the whole pad width (not only right) and produce a left click.  
  
It is very tricky and i tried to explain it in the line comments...
Maybe `{first, second, third}_zone_button` or `zone_{1, 2, 3}_button` since the number describe the zone, not the button.

Reply:
+1 for {first, second, third}_zone_button